ngraph/frontend/paddlepaddle/src/op/elementwise_ops.cpp

```diff
@@ -15,31 +15,67 @@ NamedOutputs elementwise_ops(const NodeContext& node) {
     auto x = node.get_ng_input("X");
     auto y = node.get_ng_input("Y");
 
-    auto axis = node.get_attribute<int>("axis");
-
-    PDPD_OP_VALIDATION_CHECK(node, x.get_partial_shape().rank().is_static(), "elementwise_ops: X rank must be static!");
-    PDPD_OP_VALIDATION_CHECK(node, y.get_partial_shape().rank().is_static(), "elementwise_ops: Y rank must be static!");
-    int64_t x_rank = x.get_partial_shape().rank().get_length();
-    int64_t y_rank = y.get_partial_shape().rank().get_length();
-
-    if ((axis == -1) || (axis == x_rank - 1) || (x_rank == y_rank)) {
+    auto axis_value = node.get_attribute<int>("axis");
+    if (axis_value == -1)
         return node.default_single_output_mapping({std::make_shared<T>(x, y)}, {"Out"});
-    } else {
-        // This broadcast can be implemented by either ngraph::Reshape or
-        // ngraph::Broadcast. Since PDPD implicates y_shape is a subsequence of
-        // x_shape starting from axis, to use ngraph::Reshape like Paddle2ONNX,
-        // which is more friendly to PnP.
-        auto broadcast_shape = std::vector<int64_t>(x_rank, 1);
-        PartialShape y_shape = y.get_partial_shape();
-        int32_t i = 0;
-        for (auto it = y_shape.begin(); it != y_shape.end(); ++i, ++it)
-            broadcast_shape[axis + i] = (*it).get_length();
-
-        auto reshape_node = ngraph::opset6::Constant::create(ngraph::element::i64,
-                                                             ngraph::Shape{broadcast_shape.size()},
-                                                             broadcast_shape);
-        auto y_node = std::make_shared<ngraph::opset6::Reshape>(y, reshape_node, false);
-        return node.default_single_output_mapping({std::make_shared<T>(x, y_node)}, {"Out"});
+    else {
+        auto axis = ngraph::opset6::Constant::create(ngraph::element::i64, ov::Shape{}, {axis_value});
+
+        auto x_shape = std::make_shared<ngraph::opset6::ShapeOf>(x);
```

Comment:
We'd better to use the default_opset instead of opset6, [the sample](https://github.com/openvinotoolkit/openvino/blob/2ed4e9c05f11911b55529ecf86c5c4f457c7bf54/ngraph/frontend/paddlepaddle/src/op/cumsum.cpp#L7)